ccpp/data/GFS_typedefs.meta

```diff
@@ -2411,6 +2457,126 @@
   type = real
   kind = kind_phys
   active = (control_for_microphysics_scheme == identifier_for_thompson_microphysics_scheme .and. flag_for_aerosol_physics)
+[emdust]
+  standard_name = emission_of_dust_for_smoke
+  long_name = emission of dust for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[emseas]
+  standard_name = emission_of_seas_for_smoke
+  long_name = emission of seas for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[emanoc]
+  standard_name = emission_of_anoc_for_thompson_mp
```

Comment:
How about `surface_anthropogenic_organic_carbon_emission`? I had no idea what "anoc" meant without looking in code comments. Why does it have to say "for_thompson_mp"? Couldn't it be used in a different scheme?